src/main/java/com/google/sps/service/AppConverter.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.service;
+
+import com.google.sps.models.App;
+import com.google.appengine.api.datastore.Entity;
+import java.util.ArrayList;
+import java.util.List;
+
+class AppConverter {
+    
+    public List<App> convertToApp(ArrayList<Entity> appEntities) {
```

Comment:
We should also make the input a List


`public List<App> convertToApp(List<Entity> appEntities) {`